### PR TITLE
Admin_guide: Refactor the Admin guide

### PR DESCRIPTION
**Pull request by poornimag**
Signed-off-by: Poornima G <pgurusid@redhat.com>

### CODE REVIEW

File: Administrator Guide/index.md [hunk 1]
```diff
@@ -1,74 +1,85 @@
...
-21.  Appendices
-
-	*  [Troubleshooting](./Troubleshooting.md)
-	*  [Network Configurations Techniques](./Network Configurations Techniques.md)
-	*  [Performance Testing](./Performance Testing.md)
+1.  Introduction
+	*  [About GlusterFS](../overview.md)
+	*  [Architecture](./architecture.md)
+
+2.  [Install Guide](../Installation-Guide/install.md)
```
Comment (joejulian) [inline Administrator Guide/index.md]: What about putting Prerequisites, Install Guide, and some yet-to-be-written guide to choosing the right cluster topology under "Installation" or "Getting Started" or something like that? If it was the latter, we could even move "Quick Start" under it.

File: Administrator Guide/index.md [hunk 2]
```diff
@@ -1,74 +1,85 @@
...
+
+3.  [Quick Start](../Quick-Start-Guide/Quickstart.md)
+
+4.  [Prerequisites](./prerequisites.md)
+
+5.  [Storage Pool Setup and Scaling](./storage_pool_setup_and_scaling.md)
```
Comment (joejulian) [inline Administrator Guide/index.md]: This needs retitled. I thought it was a design document until I looked at the text. How about "Trusted Pool Creation and Maintenance" or "Server Peering Administration" or something like that?  Admittedly, after reading the text I completely get this title and am having my own trouble coming up with a title that I think could be understood without context.

**Comment by humblec** [comment]
@poornimag can you please address @joejulian 's comment ? thanks for the PR

**Comment by prashanthpai** [comment]
This repo had it's git history re-written on 19 May 2016. Please delete your existing fork and create a fresh fork. You will have to re-send the PR against the latest repo.

**Pull request closed without merging**